app/controllers/checkouts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CheckoutsController < ApplicationController
+  before_action :require_login, :allow_anyone
```

Comment:
What does `allow_anyone` do and how is it different from not having that declared at all?